src/presentation/connect/spend.tsx

```diff
@@ -121,18 +121,18 @@ const ConnectSpend: React.FC<WithConnectDataProps> = ({ connectData }) => {
           </div>
         </>
       ) : (
-        <>
+        <div className="flex flex-col justify-center p-2 align-middle">
           <h1 className="mt-8 text-lg font-medium">Oops, Something went wrong...</h1>
-          <p className="font-small mt-4 text-sm">{error}</p>
+          <span className="max-w-xs mr-2 font-light">{error}</span>
           <img className="mx-auto my-10" src="/assets/images/cross.svg" alt="error" />
           <Button
             className="w-36 container mx-auto mt-10"
             onClick={handleUnlockModalOpen}
             textBase={true}
           >
-            Unlock
+            Retry
```

Comment:
why retry? this modal i clearly to unlock the seed

Reply:
This button is shown only in case of error. "Unlock" sounds weird on error page IMO. We use "Accept" btn to unlock.

Reply:
mmm,  this page could be shown even the first time without an error (ie. user cancel the original modal we pop up before attempting the first unlock etc..)

Reply:
please revert